source/index.html.md

```diff
@@ -630,6 +630,27 @@ object][ApplePay-PaymentToken] for more information.
 </p>
 
 
+##### Method: `samsungpay`
+
+Samsung Pay provides the payment details as either JWE or JWE/JWS JOSE objects.
+Currently only the JWE is accepted.
+
+<dl class="dl-vertical">
+  <dt>samsungpay[jwt]</dt>
+  <dd>
+    <a href="https://tools.ietf.org/html/rfc7516#section-3.1"><span class="type">Compact serialized JWE token</span></a>
+    <br>
+    Complete JWE token as received from Samsung Pay.
+  </dd>
+  <dt>samsungpay[cek]</dt>
```

Comment:
If we eventually support JWE/JWS, will it still make sense to accept `cek` as the parameter?

Reply:
Yes. Unless we ask for their private key.